docker/run_celery.bash

```diff
@@ -3,11 +3,7 @@ set -e
 
 source /etc/profile
 
-CELERYD_TASK_SOFT_TIME_LIMIT="${CELERYD_TASK_SOFT_TIME_LIMIT:-$((15*60))}"
```

Comment:
@jnm, is there something equivalent elsewhere to allow time for exception handling before the hard time limit interrupts the process?